onnxruntime/core/providers/openvino/openvino_graph.cc

```diff
@@ -0,0 +1,464 @@
+  }
+
+  // Prepare ModelProto Input to Python
+  PyObject* pFileName = PyByteArray_FromStringAndSize(
+      onnx_model.c_str(), onnx_model.size());
+  PyObject* pArgs = PyTuple_New(1);
+  PyTuple_SetItem(pArgs, 0, pFileName);
+
+  PyObject* pOutputTuple = NULL;
+
+  // Call the Python function
+  pOutputTuple = PyObject_CallObject(pFunc, pArgs);
+
+  if (pOutputTuple == NULL || !PyTuple_CheckExact(pOutputTuple)) {
+    throw "Python function call failure";
+  }
+
+  // Retrieve the weights byte array
+  PyObject* pArg1 = PyTuple_GetItem(pOutputTuple, 0);
+  PyObject* pWeights = PyByteArray_FromObject(pArg1);
+  const char* weights_bytes = PyByteArray_AsString(pWeights);
+  unsigned long weights_size = PyByteArray_Size(pWeights);
+  std::string weights_string(weights_bytes, weights_size);
+  openvino_bin = weights_string;
+
+  // Retrieve the xml string
+  PyObject* pArg2 = PyTuple_GetItem(pOutputTuple, 1);
+  PyObject* pXML = PyObject_Repr(pArg2);
+  openvino_xml = PyUnicode_AsUTF8(pXML);
+
+  //    Py_DECREF(pXML);
+  //    Py_DECREF(pArg2);
+  //    Py_DECREF(pWeights);
+  //    Py_DECREF(pArg1);
+  //    Py_DECREF(pOutputTuple);
+  //    Py_DECREF(pArgs);
+  //    Py_DECREF(pFunc);
+  //    Py_DECREF(pModule);
+  //    Py_Finalize();
+}
+
+std::shared_ptr<InferenceEngine::CNNNetwork> OpenVINOGraph::BuildOpenVINONetworkWithMO() {
+
+  const auto& attributes = fused_node_->GetAttributes();
+  std::string xml_string = attributes.at("xml_str").s();
+  std::string weights_string = attributes.at("weights_str").s();
+  InferenceEngine::TBlob<uint8_t>::Ptr weightsPtr(
+      new InferenceEngine::TBlob<uint8_t>(InferenceEngine::Precision::U8,
+                                          InferenceEngine::Layout::C, {weights_string.size()}));
+  weightsPtr->allocate();
+
+  std::memcpy(weightsPtr->buffer(), (void*)weights_string.c_str(), weights_string.size());
+
+  InferenceEngine::CNNNetReader networkReader;
+  networkReader.ReadNetwork((const char*)xml_string.c_str(), xml_string.size());
+  networkReader.SetWeights(weightsPtr);
+
+  return std::make_shared<InferenceEngine::CNNNetwork>(networkReader.getNetwork());
+}
+
+std::vector<InferenceEngine::InferRequest::Ptr> OpenVINOGraph::GetExecutableHandle(
+    std::shared_ptr<InferenceEngine::CNNNetwork> network,
+    const std::string& device, InferenceEngine::Precision precision) {
+  // TODO: make this better
+
+  precision = InferenceEngine::Precision::FP32;
+
+  // Load Plugin for inference engine
+  std::cout << "[OpenVINO-EP]Loading plugin" << std::endl;
+
+  std::vector<std::string> plugin_path = GetEnvLdLibraryPath();
+  plugin_path.push_back("");
+  InferenceEngine::InferencePlugin plugin = InferenceEngine::PluginDispatcher(
+                                                plugin_path)
+                                                .getPluginByDevice(device);
+
+  // Configure input & output
+  // Prepare input blobs
+  std::cout << "[OpenVINO-EP]Preparing input blobs" << std::endl;
```

Comment:
are these cout statements leftover from debugging?
better to use logging mechanisms and enable at verbose level?

Reply:
Agreed. Logging mechanism with verbose option is being implemented, that will remove these statements.

Reply:
Removed all the cout/cerr statements statements. Will incorporate logging eventually.